### ISSUE TITLE
Multiseries should allow mixing cards broken out by DateTime and numeric timestamps

### ISSUE THREAD

**Issue by tlrobinson** [metabase/metabase#2811]
This logic needs to be tightened up: https://github.com/metabase/metabase/blob/3bf28cd094c9b2516c68ed265abd3132f75a7e77/frontend/src/metabase/visualizations/components/LineAreaBarChart.jsx#L277-L284

A timestamp field is both a date and numeric so we shouldn't be enforcing the 2nd condition in the case of dates.

**Comment by tlrobinson** [comment]
Alternatively `isNumeric` shouldn't return true for `special_type` == `timestamp_milliseconds` or `timestamp_seconds`

**Comment by attekei** [comment]
My friend reported this too after trying to compose two questions where the other contained iso8601 datetimes and the other unix msec timestamps. A frontend error message caused by this:

```
Error getting setting graph.metrics TypeError: Cannot read property 'length' of null
    at t.isDimensionMetricMetric (https://dashboard.slush.org/app/dist/app-main.bundle.js?6dbbfe0722d899c239ab:92:14683)
    at b (https://dashboard.slush.org/app/dist/app-main.bundle.js?6dbbfe0722d899c239ab:92:12141)
    at l (https://dashboard.slush.org/app/dist/app-main.bundle.js?6dbbfe0722d899c239ab:101:20920)
    at a (https://dashboard.slush.org/app/dist/app-main.bundle.js?6dbbfe0722d899c239ab:101:20565)
    at Object.getDefault (https://dashboard.slush.org/app/dist/app-main.bundle.js?6dbbfe0722d899c239ab:123:26938)
    at h (https://dashboard.slush.org/app/dist/app-main.bundle.js?6dbbfe0722d899c239ab:101:22581)
    at g (https://dashboard.slush.org/app/dist/app-main.bundle.js?6dbbfe0722d899c239ab:101:22883)
    at Function.value (https://dashboard.slush.org/app/dist/app-main.bundle.js?6dbbfe0722d899c239ab:102:24409)
    at https://dashboard.slush.org/app/dist/app-main.bundle.js?6dbbfe0722d899c239ab:135:15225
    at Array.filter (native)
```

**Comment by attekei** [comment]
Addressed by #5546.
